components/Profile/ProfileLoans.tsx

```diff
@@ -10,13 +10,15 @@ type ProfileLoansProps = {
 export function ProfileLoans({ address, loans }: ProfileLoansProps) {
   return (
     <TwelveColumn>
-      {loans.map((loan) => (
-        <LoanCard
-          key={loan.id.toString()}
-          loan={loan}
-          selectedAddress={address}
-        />
-      ))}
+      {loans
+        .filter((loan) => !loan.closed)
```

Comment:
we're doing this client-side rather than at the query because `ProfileHeader` needs to see all the loans